src/paths/index/components/changes/ReleaseNotesItem.js

```diff
@@ -139,6 +140,11 @@ export default class ReleaseNotesItem extends LitElement {
     constructor() {
         super();
 
+        // Configure marked renderer to treat backticks as bold
+        const renderer = new marked.Renderer();
+        renderer.codespan = (text) => `<strong>${text}</strong>`;
+        marked.setOptions({ renderer });
+
```

Comment:
In my test, not defining this actually properly defaults to code formatting which is what I expected for `inline code` like on GitHub.
```suggestion
```